internal/git/validate.go

```
@@ -17,23 +17,23 @@ var (
 	// - Cannot end with '.lock'
 	// Using multiple regexes instead of negative lookahead
 	branchStartDotRegex = regexp.MustCompile(`^\.`)
-	doubleDotRegex     = regexp.MustCompile(`\.\.`)
-	endSlashRegex      = regexp.MustCompile(`/$`)
-	endLockRegex       = regexp.MustCompile(`\.lock$`)
+	doubleDotRegex      = regexp.MustCompile(`\.\.`)
```

golangci-lint (1.62.2)

var `doubleDotRegex` is unused

(unused)